views/dashboard.handlebars

```diff
@@ -1,18 +1,22 @@
-<div>
-    <h3>Post</h3>
-
-<div>
-
-    <p>some text</p>
-
-    <ul>
-        {{#each user.post as |post|}}
-        <li>
-            <h4> {{post.book_title}}</h4>
-            <p> {{post.book_author}}</p>
-            <p> {{post.rating}}</p>
-        </li>
-{{/each}}
-    </ul>
+<div class="dashContainer">
+    <div class="textContainer">
+        <!-- This will be for the logged in user's personal dashboard -->
+        <p>Welcome,</p>
+        <h1> {{book.user.username}} </h1>
+    </div>
+    <div class="bookContainer">
+        {{#each book as |book|}}
+        <div class="bookLinks">
+            <!-- Book image redirect to add post if none exists, or view post if there is one for this book -->
+            <a href="{{book.post_id}}">
```

Comment:
<a href="post/{{book.post.id}}>
will need the post/ before the mustaches. and not 100% sure but it might also be book.post.id instead of book.post_id.
We can try both